### DISCUSSION TITLE
Responses param doesn't work on MacOS

### DISCUSSION THREAD

**Discussion by AlexanderFarkas** [fastapi/fastapi#6555] [Questions] [answered]
### First Check

- [X] I added a very descriptive title to this issue.
- [X] I used the GitHub search to find a similar issue and didn't find it.
- [X] I searched the FastAPI documentation, with the integrated search.
- [X] I already searched in Google "How to X in FastAPI" and didn't find any information.
- [X] I already read and followed all the tutorial in the docs and didn't find an answer.
- [X] I already checked if it is not related to FastAPI but to [Pydantic](https://github.com/samuelcolvin/pydantic).
- [X] I already checked if it is not related to FastAPI but to [Swagger UI](https://github.com/swagger-api/swagger-ui).
- [X] I already checked if it is not related to FastAPI but to [ReDoc](https://github.com/Redocly/redoc).

### Commit to Help

- [X] I commit to help with one of those options 👆

### Example Code

```python
from fastapi import FastAPI
app = FastAPI(responses={422: {"e": "ValidationError", "e2": "Helpme"}})

@app.post("/")
def random_route():
    return "help me"
```

### Description

Open swagger/redoc docs.
Open post /.
I expect to see my schema  in 422 docs.
But it only states "Validation error" without any schema rendered

### Operating System

macOS

### FastAPI Version

0.68.0

### Python Version

3.9.5

**Comment by YuriiMotov** [comment, marked as answer]
Read Docs: https://fastapi.tiangolo.com/advanced/additional-responses/
